content/source/docs/enterprise/admin/upgrades.html.md

```diff
@@ -26,3 +51,36 @@ version. [Learn more about availability during upgrades here](../system-overview
     "Check Now" button; the new version should be recognized.
 1. Click "View Update".
 1. Review the release notes and then click "Install Update".
+
+### Replicated Command Line Interface
+
+1. Login to a command line terminal of the Terraform Enterprise host machine.
+1. Execute the command below and take note on the location from the property `AirgapPackagePath` to upload the desired airgap packages into on the Terraform Enterprise host machine:
+   
+    ```
+    replicatedctl params export | grep AirgapPackagePath
+    ```
+
+1. Execute the command below to fetch the versions of Terraform Enterprise from the uploaded airgap packages.
+   
+    ```
+    replicatedctl app-release ls --fetch
+    ```
+
+1. Execute the command below to list the available versions of airgap packages for the upgrade:
```

Comment:
Is this step needed? I believe the `ls --fetch` step above this will print the available packages after it updates them.

Reply:
@sudomateo It seems to print too many versions from what I have tested, including those in between the versions even without the actual airgap files existing in the directory, that's probably the reason.

You could test this and let me know if you see something different

Reply:
That's fair! Works for me.